src/test/java/io/confluent/connect/jdbc/source/TableMonitorThreadTest.java

```diff
@@ -198,6 +278,19 @@ public void execute() {
     };
   }
 
+  protected Op checkTableIds(final TableId...expectedTables) {
+    return new Op() {
+      @Override
+      public void execute() {
+        List<TableId> expectedTableIds = new ArrayList<>();
+        for (TableId expectedTable: expectedTables) {
+          expectedTableIds.add(expectedTable);
+        }
+        assertEquals(expectedTableIds, tableMonitorThread.tables());
+      }
+    };
```

Comment:
Could use a lambda here:
```
protected Op checkTableIds(final TableId...expectedTables) {
   return ()-> {
        List<TableId> expectedTableIds = Arrays.asList(expectedTables);
        assertEquals(expectedTableIds, tableMonitorThread.tables());
   };
```
Also, this check also verifies the IDs are in the same order. Is that sufficient?

Finally, why does this return a lamba in the first place? Every place this method is called, you're immediately calling `execute()`. Why not just use a helper method that performs the assertion?

```
protected void checkTableIds(final TableId...expectedTables) {
    List<TableId> expectedTableIds = Arrays.asList(expectedTables);
    assertEquals(expectedTableIds, tableMonitorThread.tables());
}
```